ql/src/java/org/apache/hadoop/hive/ql/plan/LoadFileDesc.java

```diff
@@ -152,4 +153,12 @@ public String getMoveTaskId() {
   public void setMoveTaskId(String moveTaskId) {
     this.moveTaskId = moveTaskId;
   }
+
+  public boolean getIsOverwrite() {
+    return isOverwrite;
+  }
+
+  public void setIsOverwrite(boolean isOverwrite) {
+    this.isOverwrite = isOverwrite;
+  }
```

Comment:
It is better to name the getters/setters for boolean members like `getOverwrite()` and `setOverwrite`.

https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter